fava/ext/portfolio_list/__init__.py

```diff
@@ -0,0 +1,154 @@
+"""Portfolio list extension for Fava.
+
+This is a simple example of Fava's extension reports system.
+"""
+import os
+import re
+
+from beancount.core.data import iter_entry_dates, Open
+from beancount.core.number import ZERO, Decimal
+
+from fava.ext import FavaExtensionBase
+from fava.template_filters import cost_or_value
+from fava.core.tree import Tree
+from fava.core.helpers import FavaAPIException
+
+
+class PortfolioList(FavaExtensionBase):  # pragma: no cover
+    """Sample Extension Report that just prints out an Portfolio List.
+    """
+
+    # pylint: disable=eval-used
+    def __init__(self, ledger, extension_config):
+        super().__init__(ledger, extension_config)
+        self.config = eval(extension_config, {}, {})
+        self.report_title = "Portfolio List"
+
+    def report_template(self):
```

Comment:
This code should be moved to the extensions core module

Reply:
Done, report_title attr is still there but it's now optional.